atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -173,6 +173,29 @@ def set_media_types(self, extra, worker_metadatas):
                 extra['image']['media_types'] = json.loads(annotations['media-types'])
                 return
 
+    def set_manifest_list_info(self, extra, worker_metadatas):
+        manifest_list_digests = self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY)
+        if manifest_list_digests:
+            index = {}
+            index['tags'] = [image.tag for image in self.workflow.tag_conf.images]
+            repositories = self.workflow.build_result.annotations['repositories']['unique']
+            repo = ImageName.parse(repositories[0]).repo
```

Comment:
This should be: `repo = ImageName.parse(repositories[0]).to_str(registry=False, tag=False)`.

Reply:
Fixed.